indico/modules/rb_new/__init__.py

```diff
@@ -40,6 +44,27 @@ def _sidemenu_items(sender, event, **kwargs):
                            icon='location')
 
 
+@signals.users.merged.connect
+def _merge_users(target, source, **kwargs):
+    BlockingPrincipal.merge_users(target, source, 'blocking')
```

Comment:
move down to the other principal merge (or the other one up here)?

Reply:
Well, my logic was actually to group things together: first blockings, then reservations and finally rooms.